### ISSUE TITLE
ASHLAR

### ISSUE THREAD

**Issue by ssahoo00** [labsyspharm/ashlar#154]

**Comment by jmuhlich** [comment]
Yes, but only if A) you have individual field-of-view tile images (**not** a pre-stitched mosaic image), and B) the H&E and IF images were acquired on the same microscope and pixel resolution. What microscope did you use to acquire the images?

**Comment by ssahoo00** [comment]
Dear Jeremy,

Thank you very much for the mail and notification.

Here are the details about images:

It is Keyence BZ-X800.

The H+E data will be .VSI files images, these are processed on the fly so there is no unstitched data.

Best Regards,
Sudhakar
________________________________
